csharp/FileEncryptionProperties.cs

```diff
@@ -6,6 +6,9 @@ namespace ParquetSharp
     /// <summary>
     /// Properties related to encrypting a parquet file.
     /// </summary>
+    /// <remarks>
+    /// This class is used to retrieve encryption properties from a parquet file.
```

Comment:
This makes it sound like you can get the encryption properties from a file after it has been written, but that's not correct. This class is only used when writing a Parquet file. I think this remark could just be removed.